app/facades/dashboard_show_facade.rb

```diff
@@ -17,18 +17,18 @@ def recommended_songs
     songs = @user.top_songs(5).map(&:spotify_id).join(',')
     @recommended_songs ||= recommended_service.get_recommendations(songs)
   end
-	
+
 	def recommended_api_url
```

Comment:
Is there a reason this is indented? I will resolve for now and change it with the next PR.